core/block_dragger.js

```diff
@@ -179,8 +179,11 @@ Blockly.BlockDragger.prototype.startBlockDrag = function(currentDragDeltaXY) {
   // surface.
   this.draggingBlock_.moveToDragSurface_();
 
-  if (this.workspace_.toolbox_) {
-    this.workspace_.toolbox_.addDeleteStyle();
+  var toolbox = this.workspace_.getToolbox();
```

Comment:
We changed the API for toolbox style from `addDeleteStyle` to `addStyle(style)`.  We use the new version in the bubble dragger, so needed to delete the old version and use the new one here as well.